assets/src/styles/blocks/Columns.scss

```diff
@@ -204,26 +208,30 @@
         }
 
         @include large-and-up {
+          --block-columns--step-number-- {
+            background-color: $dark-blue;
```

Comment:
Don't you usually prefer to use `background` instead of `background-color` for more flexibility?

Reply:
Changed it :+1: